consensus/validation.go

```diff
@@ -715,6 +715,8 @@ func validateV2FileContracts(ms *MidState, txn types.V2Transaction) error {
 
 	validateContract := func(fc types.V2FileContract, renewal bool) error {
 		switch {
+		case fc.Filesize > fc.Capacity:
```

Comment:
Could it be that we are missing `fc.Filesize % SectorSize == 0` checks? Same for capacity?

Reply:
`SectorSize` is not a consensus concept, only an RHP one.

Reply:
@lukechampine what about `SegmentSize`?